api/test/generate-data/factories/list_factory.js

```diff
@@ -5,10 +5,10 @@ let faker = require('faker/locale/en');
 
 const factoryName = List.modelName;
 
-const ListArray1 = require('../../../migrations_data/lists.js');
-const ListArray2 = require('../../../migrations_data/new_list_items.js');
-const ListArray3 = require('../../../migrations_data/newProjectPhaseListItems.js');
-const ListArray4 = require('../../../migrations_data/regionList.js');
+const ListArray1 = require('../../../../migrations_data/lists.js');
```

Comment:
I find it more intuitive to use full paths vs relative paths, except when references directly relate to a single encapsulated concept or component (eg. CoolList imports './CoolListItem.js', but everything imports CoolList with a full references.

Reply:
yeah this is a good point. We could work on doing a refactor, would look cleaner especially with how many folders we need to traverse.